### ISSUE TITLE
Highlight when `TrustSet` freezes or unfreezes a trust line in the "Simple" view 

### ISSUE THREAD

**Issue by ckniffen** [ripple/explorer#393]
Use `tfSetFreeze` and `tfClearFreeze` to highlight on the left of the "SimpleTab" view to show "Frozen" or "Unfrozen" respecitivly.

Wording may need updating.  Thoughts @mduo13 @bugsbunnies @nataliacebotarii

# Example Transactions:

- [Frozen]( https://livenet.xrpl.org/transactions/9861901B6C5336E2EC46E54866EB1E3B42CAD4A9EBB7F8AA47548C7891E16F2D/detailed)
- [Unfrozen](https://livenet.xrpl.org/transactions/A536E59E035FA290F02A6EB40FAE04FBCF8381E28ED92140441F02B7B68C44D3/detailed)

**Comment by bugsbunnies** [comment]
The left of the Simple tab view right now has a label and a value, I think we can add "Frozen" or "Unfrozen" as the value but what should the label say? Flags? @ckniffen

**Comment by mvadari** [comment]
`Trustline Status`? It's not covering all flags

**Comment by ckniffen** [comment]
> `Trustline Status`? It's not covering all flags

The flags do show up but are buried on the details page.  I feel like freeze use-cases should have greater visibility.

**Comment by mDuo13** [comment]
I'm supportive; maybe it should be clear _who's_ freezing the line since depending on which side is [currently] the issuer the implications of freezing/unfreezing the line are different.

**Comment by ckniffen** [comment]
What do you by which side?  Do you mean whether the freezer is the issuer?

**Comment by ckniffen** [comment]
How would you for see the messaging being different?

**Comment by mDuo13** [comment]
I am not sure what the expected display is; as long as it's clear _who's_ doing the freezing—i.e. if the sender of the transaction is obvious from the display that's OK. 

Possibly a better display would be possible if you can identify who's the issuer on the trust line (that is, whose balance is negative from their perspective). You could add a warning/tip note if the freeze is enacted by the holder, _not_ by the issuer, because the semantics of what happens in that case are perhaps counter to what most users expect. The [Individual Freeze docs](https://xrpl.org/freezes.html#individual-freeze) have the details:

> An individual address can freeze its trust line to a financial institution. This has no effect on transactions between the institution and other users. It does, however, prevent other addresses, including [operational addresses](https://xrpl.org/issuing-and-operational-addresses.html), from sending that financial institution's tokens to the individual address. This type of individual freeze has no effect on offers.

In other words, if you freeze your own trust line then you can't get more of that token even from the token issuer's hot wallet. I think even direct transactions can't increase your balance on that trust line? But you can still send/spend it.
